watch-parties/src/main/java/com/google/sps/data/Chat.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.data;
```

Comment:
Let's keep this server work out of this PR (like we talked about at sync) and move it to its own branch.  I'd like to keep this PR focused on just getting the prototype work merged.